src/pybind/mgr/dashboard_v2/frontend/src/app/shared/components/datatable/table-key-value/table-key-value.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, Input, OnChanges } from '@angular/core';
+
+/**
+ * Display the given data in a 2 column data table. The left column
+ * shows the 'key' attribute, the right column the 'value' attribute.
+ * The data table has the following characteristics:
+ * - No header and footer is displayed
+ * - The relation of the width for the columns 'key' and 'value' is 1:3
+ * - The 'key' column is displayed in bold text
+ */
+@Component({
+  selector: 'cd-table-key-value',
+  templateUrl: './table-key-value.component.html',
+  styleUrls: ['./table-key-value.component.scss']
+})
+export class TableKeyValueComponent implements OnChanges {
+
+  private columns: Array<object> = [];
+
+  /**
+   * An array of objects to be displayed in the data table.
+   */
+  @Input() data: Array<object> = [];
+
+  /**
+   * The name of the attribute to be displayed as key.
+   * Defaults to 'key'.
+   * @type {string}
+   */
+  @Input() key = 'key';
+
+  /**
+   * The name of the attribute to be displayed as value.
+   * Defaults to 'value'.
+   * @type {string}
+   */
+  @Input() value = 'value';
+
+  constructor() {
+    this.setupColumns();
+  }
+
+  ngOnChanges(changes) {
+    this.setupColumns();
+  }
+
+  private setupColumns() {
+    this.columns = [
+      {
+        prop: this.key,
+        flexGrow: 1,
+        cellClass: 'bold'
```

Comment:
You should use the bold template here instead.

Reply:
Done